<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1a1a1a;
color: #e7e7e5</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="layout" native="true">
     <property name="maximumSize">
      <size>
       <width>1094</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
background-color: #202122;
}

QTextEdit {
border: 1px solid #353738;
border-radius: 5px;
}

QTextEdit:focus {
border: 1px solid #dc5049;
}

QComboBox {
border: 1px solid #353738;
border-radius: 5px;
}

QComboBox:focus {
border: 1px solid #dc5049;
}</string>
     </property>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Control System Work Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Приоритеты&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>1091</width>
        <height>661</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_roles">
       <attribute name="title">
        <string>Просмотр</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>1090</width>
          <height>521</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>220</red>
              <green>80</green>
              <blue>73</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>220</red>
              <green>80</green>
              <blue>73</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>32</red>
              <green>33</green>
              <blue>34</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>231</red>
              <green>231</green>
              <blue>229</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #353738</string>
        </property>
       </widget>
       <widget class="QLabel" name="searchLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>351</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Control System Work Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5a7aa;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Поиск по названию и описанию&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="searchEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="searchButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCommandLinkButton {
border-radius: 5px;
}

QCommandLinkButton:hover {
background-color: #dc5049;
}</string>
        </property>
        <property name="text">
         <string>Найти</string>
        </property>
        <property name="description">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_create_role">
       <attribute name="title">
        <string>Добавить</string>
       </attribute>
       <widget class="QCommandLinkButton" name="createButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>450</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCommandLinkButton {
border-radius: 5px;
}

QCommandLinkButton:hover {
background-color: #dc5049;
}</string>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="description">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="descLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>180</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Control System Work Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5a7aa;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Описание&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="titleLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Control System Work Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5a7aa;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Название&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="titleEdit">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>120</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QTextEdit" name="descEdit">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>210</y>
          <width>561</width>
          <height>171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
